test_treematcher.py

```diff
@@ -26,13 +26,13 @@ def test_ete_params(self):
             """
 
         pattern0 = TreePattern(pattern0)
-        pattern1 = TreePattern(pattern1, format=8, quoted_node_names=False)
-        pattern2 = TreePattern(pattern2, format=8, quoted_node_names=True)
+        pattern1 = TreePattern(pattern1)
+        pattern2 = TreePattern(pattern2)
 
         tree = Tree("(hello,(1,2,3)kk)pasa:1;", format=1)
-        match0 = pattern0.find_match(tree, None, maxhits=1)
```

Comment:
No need for the second argument anymore. Also thinking if find_match-maxhits should default to None rather than 1...